app/controllers/collection_associations_controller.rb

```diff
@@ -57,4 +79,18 @@ def add_high_level_breadcrumbs
 
     add_breadcrumb 'Collection Associations', variable_collection_associations_path(params[:variable_id])
   end
+
+  def log_issues(response)
+    response.body.select { |assocation_response| assocation_response.fetch('warnings', []).any? }.each do |warnings|
+      warnings.each do |warning|
+        Rails.logger.error "Variable Assocation [warning]: #{warning}"
```

Comment:
I don't know what the warnings or errors look like, but does this provide sufficient information (i.e. the variable and/or collection concept ids)?

Reply:
Yes, it does.